ocp_utilities/must_gather.py

```diff
@@ -51,3 +54,37 @@ def run_must_gather(
         flag_string = "".join([f" --{flag_name}" for flag_name in flag_names])
         base_command += f" {flag_string}"
     return run_command(command=shlex.split(base_command), check=False)[1]
+
+
+def collect_must_gather(must_gather_output_dir, cluster_name, product_name, kubeconfig_path=None):
+    """
+    Run must-gather command with an option to create target directory using must_gather_output_dir, cluster_name, product_name.
+    (This function can be used for running must-gather incase any product installation fails.)
```

Comment:
args are described in the `Args` part, no need to repeat here.
please follow doscrting formatting (e.g new line in missing after 1st line and how it should be written)